n2y/converter.py

```diff
@@ -133,8 +135,15 @@ def get_children(self):
                 previous_child_type = child['type']
 
 
+class Equation():
```

Comment:
Does `Equation` or `EquationBlock` need to be added to the readme?

Reply:
Just `EquationBlock`, I've deleted `Equation`